org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java

```diff
@@ -76,7 +76,7 @@ public static void cleanTempDirectory() {
 			@Override
 			public FileVisitResult visitFile(Path path, BasicFileAttributes attrs) throws IOException {
 				if (pathMatcher.matches(path)) {
-					String tempPath = (path.toString().replaceAll(location + "/", "")).toLowerCase();
+					String tempPath = StringHandler.toCamelCase((path.toString().replaceAll(location + "/", "")).toLowerCase());
```

Comment:
Why is the camel case necessary? Why don't you take the file name, as is?

Reply:
(without the `toLowerCase()`, of course)

Reply:
So if  you only take `path.toString().replaceAll(location + "/", "")`, why isn't it working? What is the observed behavior?